draft-ietf-ppm-dap.md

```diff
@@ -2391,7 +2391,10 @@ If all of the above checks succeed, the input share is valid.
 The Helper handles the aggregation job initialization synchronously:
 
 ~~~ http
-PUT /helper/tasks/8BY0RzZMzxvA46_8ymhzycOB9krN-QIGYvg_RsByGec/aggregation_jobs/lc7aUeGpdSNosNlh-UZhKA
+========== NOTE: '\' line wrapping per RFC 8792 ===========
+
+PUT /helper/tasks/8BY0RzZMzxvA46_8ymhzycOB9krN-QIGYvg_RsByGec/\
+aggregation_jobs/lc7aUeGpdSNosNlh-UZhKA
```

Comment:
When wrapping, you can add a small indent to the continuation line (which the 8792 processing skips).  For human readers, that's handy.